packages/grid/_modules_/grid/components/styled-wrappers/GridRootStyles.ts

```diff
@@ -10,6 +10,7 @@ export const useStyles = makeStyles(
     const gridStyle: { root: any } = {
       root: {
         flex: 1,
+        touchAction: 'none',
```

Comment:
We can't prevent scroll on the whole grid. It should only be applied on the resizing area.

Reply:
For instance, you can't scroll starting from the pagination.

Reply:
I moved it to the cell separator, this should do the trick. I tested it on IOS (Chrome and Safari) it is working. It should also work on Android.